keras_hub/src/models/pali_gemma/pali_gemma_presets.py

```diff
@@ -66,9 +68,10 @@
             "path": "pali_gemma2",
             "model_card": "https://www.kaggle.com/models/google/paligemma-2",
         },
-        "kaggle_handle": "kaggle://keras/paligemma2/keras/pali_gemma2_3b_ft_docci_448/1",
+        # TODO: Rename `pali_gemma_2_ft_docci_3b_448` to `pali_gemma2_ft_docci_3b_448`
+        "kaggle_handle": "kaggle://keras/paligemma2/keras/pali_gemma_2_ft_docci_3b_448/1",
```

Comment:
Does kaggle handle redirects with a rename? @divyashreepathihalli

Reply:
This is probably important to resolve before we merge this, otherwise we need to start thinking about the ordering of things.

Reply:
Unable to just rename, we would need to reupload.